From e908457423cf9b8c0ab70ae86cf6a5edd1584600 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 2 Jul 2024 14:04:13 +0200
Subject: [PATCH 0997/1695] dts: stm32: activate HSI calibration on
 STM32MP215f-dk board

Default enable HSI (High Speed Internal oscillator) calibration. This
internal oscillator tends to derive over time so calibration is needed.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git da1b3088122a6d9d72cf6fa5fdb9504830ce1f31 dts: stm32: activate HSI calibration on STM32MP215f-dk board]
Change-Id: I71483276f1c03959c72d1beba138279a122f4f9a
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/390831
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 1464e0c8b..fc5b2c8a9 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -41,6 +41,10 @@
 	status = "okay";
 };
 
+&hsi_calibration {
+	status = "okay";
+};
+
 &i2c3 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c3_pins_a>;
-- 
2.47.2

